subprojects/core-api/src/main/java/org/gradle/api/initialization/Settings.java

```diff
@@ -359,31 +359,27 @@ default void includeFlat(String... projectNames) {
      *
      * @since 6.8
      */
-    @Incubating
     void dependencyResolutionManagement(Action<? super DependencyResolutionManagement> dependencyResolutionConfiguration);
 
     /**
      * Returns the dependency resolution management handler.
      *
      * @since 6.8
      */
-    @Incubating
     DependencyResolutionManagement getDependencyResolutionManagement();
 
     /**
      * Configures toolchain management.
      *
      * @since 7.6
      */
-    @Incubating
```

Comment:
In theory, I agree. But since we remove the alternative in 8.0 for auto provisioning, it does not feel right to only leave an incubating option.
We will already have 7.6 with only Incubating or Deprecated choices.

Reply:
Ok.